docs/pages/documentation/3-querying-with-graql/insert-queries.md

```diff
@@ -9,9 +9,13 @@ permalink: /documentation/graql/insert-queries.html
 folder: documentation
```

Comment:
This comment is not related to this line but rather just a general note. I am wondering how "followable" the tutorial is now since we have different pages for define and insert.